docs/Install.md

```diff
@@ -4,115 +4,161 @@ Docker Compose will help to start the application locally on your computer and s
 
 Prerequisites for this are:
 
-* Git
-* Docker
-* Docker Compose
-* Recommend 8GB RAM
+    * Git
+    * Docker
+    * Docker Compose
+    * Recommend 8GB RAM
 
 **Note:** For windows user, enable WSL engine on Docker Desktop. Check [FAQ](./FAQ.md) to enable WSL
 
-as a first step you need to clone the Git Repo to your local computer, this is done by opening terminal/command prompt (or some visual git client you may have) and executing:
+As a first step you need to clone the Git Repo to your local computer, this is done by opening terminal/command prompt (or some visual git client you may have) and executing:
+
+    ```
+    $git clone https://github.com/mercedes-benz/DnA.git
 
-```
-git clone https://github.com/mercedes-benz/DnA.git
-```
+    ```
 
 Once when cloning is finalized you will have a copy of the entire repository locally (replace <`<Cloned Folder>`> with actual location on your computer) and now you can simply start the application
 
-```
-cd <<Clonned Folder>>/deployment/
+    ```
+    $cd <<Clonned Folder>>/deployment/
```

Comment:
remove the $ sign
i am not  sure, why you have changed the dokcercompose section